src/listen/websocket.rs

```diff
@@ -361,23 +446,48 @@ where
             let mut write = write_clone.lock().await;
             if let Err(e) = write.send(Message::binary([])).await {
```

Comment:
We should send a proper close message here, so the user doesn't have to explicitly interact with the stream to get proper closing behavior when the stream is done (this gets rid of the need for interacting with a `TranscriptionStream` while using the stream-driven interface.

Reply:
Resolved *I think